### ISSUE TITLE
log IAT events with Transcriptor

### ISSUE THREAD

**Issue by yoshimo** [Dragnogd/Instance-Achievement-Tracker#21]
to figure out patterns and warnings for encounters BW,LW and DBM log the fight with
https://github.com/BigWigsMods/Transcriptor/ 

All of these offer events that Transcriptor adds alongside the combat events to correlate them better and figure out if it matches.

I think you should add support for IAT on both sides like done in https://github.com/BigWigsMods/Transcriptor/blob/master/Transcriptor.lua#L1732 and related lines.
Also logging the Blizzard achievment tracking with it.
This would make it easier to see if our assumptions are correct in the future.

**Comment by Dragnogd** [comment]
Yes, This sounds like a good idea/improvement and definitely something I can at *hopefully* before the next set of raids release.

**Comment by yoshimo** [comment]
so 10.1 is there, the raid isn't yet. Any progress on this issue @Dragnogd  ? ;)

**Comment by yoshimo** [comment]
@Dragnogd  The War Within is around the corner and Cata Classic is out, how about it? :)
